.circleci/config.yml

```diff
@@ -123,6 +112,30 @@ workflows:
     jobs:
       - run-sdk-instrumentation-test
 
+  weekly:
+    triggers:
+      - schedule:
+          # at 7:00 on Monday
+          cron: 0 7 * * 1
+          filters:
+            branches:
+              only:
+                - main
+    jobs:
+      - check-gradle-dependencies
+      - run-robo-test:
+          nightly: true
+          requires:
+            - build-modules-and-instrumentation-tests
```

Comment:
nit:
maybe make it required from `run-sdk-instrumentation-test`?
to spend less resources in case of failure

Reply:
:+1: , will reverse it though, first run robo after run instrumentation tests